packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 4.24.0 - (December 13, 2022)
```

Comment:
what are the changes causing minor version bump for terra-notification-dialog

Reply:
Looks like due to these [changes](https://github.com/cerner/terra-framework/pull/1581/files#diff-fe0ecfb094b0833768b2f52e0a9c40db0348bcb4e0e7e2cf9d9b51361de63823).

Reply:
We can manually reassign these super minor doc-only or changelog-only updates as a patch change, if/when we remember to catch them. Not the worst if we miss them and they go out as minor change instead.